dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -33,6 +33,7 @@
 import com.dotmarketing.exception.DotSecurityException;
```

Comment:
I'm not sure if we have all the tests for method importSecretsAndSave covered here

Reply:
covered here : Test_Create_Secrets_Then_Export_Them_Then_Import_Then_Save